### PR TITLE
6115_Handler_Click_Event

### PR DESCRIPTION
**Pull request by dinhquang251**
Handler event click in Home UI
https://edu-redmine.framgia.vn/issues/6115

### CODE REVIEW

**Comment by dinhquang251** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_35/screen/detail/DetailActivity.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+import com.framgia.moviedb_35.R;
+
+public class DetailActivity extends AppCompatActivity {
+    public static Intent getDetailIntent(Context context){
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/screen/detail/DetailActivity.java]:
```suggestion
    public static Intent getIntent(Context context){
```

File: app/src/main/java/com/framgia/moviedb_35/screen/home/CategoriesAdapter.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
 
 public class CategoriesAdapter extends RecyclerView.Adapter<CategoriesAdapter.ViewHolder> {
     private List<Movie> mMovies;
+    private ItemClickListener mClickListener;
 
     public CategoriesAdapter(ArrayList<Movie> movies) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/screen/home/CategoriesAdapter.java]:
```suggestion
    public CategoriesAdapter(List<Movie> movies) {
```

File: app/src/main/java/com/framgia/moviedb_35/util/StringUtils.java [hunk 3]
```diff
@@ -10,6 +10,6 @@ public static String append(String... strings) {
     }
 
     public static String getImageLink(int size, String url) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/util/StringUtils.java]:
```suggestion
    public static String getImageUrl(int size, String childUrl) {
```

File: app/src/main/res/layout/activity_detail.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/image_background"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_detail.xml]:
```suggestion
        android:id="@+id/image_back_ground"
```

File: app/src/main/res/layout/activity_detail.xml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/image_avatar"
+        android:layout_width="@dimen/dp_120"
+        android:layout_height="@dimen/dp_170"
+        android:layout_marginLeft="@dimen/dp_15"
+        android:layout_marginTop="@dimen/dp_200"
+        android:scaleType="fitXY"
+        android:src="@drawable/background"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_detail.xml]:
```suggestion
        android:src="@drawable/back_ground"
```

File: app/src/main/res/layout/activity_detail.xml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        android:id="@+id/text_manufacturer_movie"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/app_name"
+        app:layout_constraintBottom_toTopOf="@+id/rating_Bar"
+        app:layout_constraintLeft_toLeftOf="@id/text_view_movie"
+        app:layout_constraintTop_toBottomOf="@+id/text_view_movie" />
+
+    <RatingBar
+        android:id="@+id/rating_Bar"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_detail.xml]:
```suggestion
        android:id="@+id/rating_bar"
```

File: app/src/main/res/layout/item_movie.xml [hunk 7]
```diff
@@ -3,6 +3,8 @@
 
     <data>
 
+        <import type="android.view.View"></import>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/item_movie.xml]: close self tag

**Comment by dinhquang251** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**